### ISSUE TITLE
Suggestion: Nokia 1 with MT6737M has kernel v4.9. How about grandppltedx?

### ISSUE THREAD

**Issue by bmndc** [almondnguyen/android_kernel_samsung_grandppltedx#5]
Hi again, I see that MT6737T and MT6737M are not much different from each other. So is it possible for kernel version 4.9 to also make it to the J2 Prime? Some news sources even tell me that Nokia upgraded the device to Android 10 Go. ;)

Source code of the Nokia 1 kernel can be found here: https://www.nokia.com/phones/en_int/opensource

**Comment by almondnguyen** [comment]
To start, sorry to break the bubble, but "see" doesn't necessary means it "is".
In fact, you cannot use MT6737M base firmware to port to this device. There are incompabilities where you need a whole BSP and quite some time to make out the difference.

Speaking of BSP,
![image](https://user-images.githubusercontent.com/42428221/155069370-f8489eff-732d-41a3-8276-735bb48dd317.png)

I can upstream to even 4.14 if I care enough to dedicate my time to.

I won't though. It took HMD over a year for Android 10 Go + Kernel 4.9-LC, and they are professionals with dedicated teams and with proper BSP and instructions from MTK itself. This device hasn't gone past a stable AOSP release yet (you can probably get the grasp of my available time and my skills). 

:)


Also, (my) problems with upstreaming kernel on this device:
- Going up to 3.18.119 by mainline upstream already breaks one a two things.
- This thing isn't some generic MTK devices that you can build out of a fresh leaked BSP. Nope, _this thing has all the Samsung shits literally everywhere_. The kernel is obviously not an exception. Forward porting those fw, drivers and stuff is way above my (current) expertise, time and mood.
- I'm not even familiar with updating _just_ MTK drivers from BSP.


TL;DR: no I won't for the near future. Too much time and effort for a lone dev doing out of hobby.

Thank you and best regards.
